--- libosip-0.8.9/doc/Makefile.am~	Thu Oct  3 11:47:16 2002
+++ libosip-0.8.9/doc/Makefile.am	Thu Oct  3 11:47:16 2002
@@ -1,12 +1,6 @@
 EXTRA_DIST = osip.sgml
 
-mandir = @mandir@/man1
-
-install-manpage: osip.1
-	$(mkinstalldirs) $(DESTDIR)$(mandir)
-	@chmod 0755 $(DESTDIR)$(mandir)
-	@echo "Installing man page: osip.1"
-	$(INSTALL_DATA) $(top_srcdir)/doc/osip.1 $(DESTDIR)$(mandir);
+man_MANS = osip.1
 
 osip.1: osip.sgml
 	docbook-to-man $< > $@
--- libosip-0.8.9/fsm/Makefile.am~	Thu Oct  3 11:48:30 2002
+++ libosip-0.8.9/fsm/Makefile.am	Thu Oct  3 11:48:30 2002
@@ -12,10 +12,10 @@
 
 
 libfsmtl_la_LDFLAGS = -version-info $(LIBOSIP_SO_VERSION) \
--release $(OSIP_RELEASE)  @FSM_LIB@  @EXTRA_LIB@ @PTHREAD_LIBS@
+-release $(OSIP_RELEASE) $(FSM_LIB) $(EXTRA_LIB)
 
 
 INCLUDES = -I$(top_srcdir)
-CFLAGS = -Wall @PTHREAD_CFLAGS@ @SIP_CFLAGS@ @SIP_FSM_FLAGS@ @SIP_EXTRA_FLAGS@
+CFLAGS = -Wall $(SIP_CFLAGS) $(SIP_FSM_FLAGS) $(SIP_EXTRA_FLAGS)
 
 
--- libosip-0.8.9/parser/Makefile.am~	Thu Oct  3 11:49:30 2002
+++ libosip-0.8.9/parser/Makefile.am	Thu Oct  3 11:49:30 2002
@@ -18,10 +18,10 @@
 
 
 libosip_la_LDFLAGS = -version-info $(LIBOSIP_SO_VERSION) \
--release $(OSIP_RELEASE) @PARSER_LIB@ @EXTRA_LIB@
+-release $(OSIP_RELEASE) $(PARSER_LIB) $(EXTRA_LIB)
 
 
 INCLUDES = -I$(top_srcdir)
-CFLAGS = -Wall @SIP_CFLAGS@ @SIP_PARSER_FLAGS@ @SIP_EXTRA_FLAGS@
+CFLAGS = -Wall $(SIP_CFLAGS) $(SIP_PARSER_FLAGS) $(SIP_EXTRA_FLAGS)
 
 
--- libosip-0.8.9/test/Makefile.am~	Thu Oct  3 11:52:42 2002
+++ libosip-0.8.9/test/Makefile.am	Thu Oct  3 11:52:42 2002
@@ -3,39 +3,34 @@
 
 noinst_PROGRAMS = torture_test torture_sdp_test turl tfrom tto tcontact tvia tcallid tcontentt trecordr troute twwwa
 INCLUDES = -I$(top_srcdir)
-CFLAGS = -Wall @SIP_CFLAGS@ @SIP_PARSER_FLAGS@ @SIP_EXTRA_FLAGS@
+CFLAGS = -Wall $(SIP_CFLAGS) $(SIP_PARSER_FLAGS) $(SIP_EXTRA_FLAGS)
 
 twwwa_SOURCES =  twwwa.c
-twwwa_LDFLAGS = -O  -Wall 
-twwwa_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+twwwa_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o   $(top_builddir)/parser/urls.o \
 $(top_builddir)/parser/port_malloc.o $(top_builddir)/parser/port_misc.o \
 $(top_builddir)/parser/hdr_wwwauthenticate.o
 
 tcontentt_SOURCES =  tcontentt.c
-tcontentt_LDFLAGS = -O  -Wall 
-tcontentt_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+tcontentt_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o   $(top_builddir)/parser/urls.o \
 $(top_builddir)/parser/port_malloc.o $(top_builddir)/parser/port_misc.o \
 $(top_builddir)/parser/hdr_contenttype.o $(top_builddir)/parser/hdr_from.o
 
 turl_SOURCES =  turls.c
-turl_LDFLAGS = -O  -Wall 
-turl_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+turl_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o \
 $(top_builddir)/parser/hdr_from.o
 
 tfrom_SOURCES =  tfrom.c
-tfrom_LDFLAGS = -O  -Wall 
-tfrom_LDADD =  @PARSER_LIB@ @EXTRA_LIB@ \
+tfrom_LDADD =  $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_headers.o   $(top_builddir)/parser/hdr_from.o
 
 tto_SOURCES =  tto.c
-tto_LDFLAGS = -O  -Wall 
-tto_LDADD = @PARSER_LIB@ @EXTRA_LIB@  \
+tto_LDADD = $(PARSER_LIB) $(EXTRA_LIB)  \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_headers.o   $(top_builddir)/parser/hdr_to.o   \
@@ -43,24 +38,21 @@
 
 
 tcontact_SOURCES =  tcontact.c
-tcontact_LDFLAGS = -O  -Wall 
-tcontact_LDADD =  @PARSER_LIB@ @EXTRA_LIB@  \
+tcontact_LDADD =  $(PARSER_LIB) $(EXTRA_LIB)  \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_headers.o   $(top_builddir)/parser/hdr_from.o \
 $(top_builddir)/parser/hdr_to.o        $(top_builddir)/parser/hdr_contact.o
 
 trecordr_SOURCES =  trecordr.c
-trecordr_LDFLAGS = -O  -Wall 
-trecordr_LDADD =  @PARSER_LIB@ @EXTRA_LIB@  \
+trecordr_LDADD =  $(PARSER_LIB) $(EXTRA_LIB)  \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_headers.o   $(top_builddir)/parser/hdr_from.o \
 $(top_builddir)/parser/hdr_to.o        $(top_builddir)/parser/hdr_recordroute.o
 
 troute_SOURCES =  troute.c
-troute_LDFLAGS = -O  -Wall 
-troute_LDADD =  @PARSER_LIB@ @EXTRA_LIB@ \
+troute_LDADD =  $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_headers.o   $(top_builddir)/parser/hdr_from.o \
@@ -68,32 +60,28 @@
 $(top_builddir)/parser/hdr_recordroute.o
 
 tvia_SOURCES =  tvia.c
-tvia_LDFLAGS = -O  -Wall 
-tvia_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+tvia_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_headers.o   $(top_builddir)/parser/hdr_via.o \
 $(top_builddir)/parser/hdr_from.o
 
 tcallid_SOURCES =  tcallid.c
-tcallid_LDFLAGS = -O  -Wall 
-tcallid_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+tcallid_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o     $(top_builddir)/parser/urls.o    \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_callid.o    $(top_builddir)/parser/hdr_from.o
 
 
 torture_sdp_test_SOURCES =  torture_sdp.c
-torture_sdp_test_LDFLAGS = -O  -Wall 
-torture_sdp_test_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+torture_sdp_test_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o   \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/sdp_rfc2327.o   $(top_builddir)/parser/sdp_accessor.o \
 $(top_builddir)/fsm/sdp_negoc.o
 
 torture_test_SOURCES =  torture.c
-torture_test_LDFLAGS = -O  -Wall 
-torture_test_LDADD = @PARSER_LIB@ @EXTRA_LIB@ \
+torture_test_LDADD = $(PARSER_LIB) $(EXTRA_LIB) \
 $(top_builddir)/parser/port_list.o   \
 $(top_builddir)/parser/port_malloc.o   $(top_builddir)/parser/port_misc.o  \
 $(top_builddir)/parser/hdr_body.o      $(top_builddir)/parser/hdr_callid.o   \
